src/pages/Player.vue

```diff
@@ -341,6 +384,44 @@ export default {
   },
   computed: {
     ...mapGetters("auth", ["isAuthenticated"]),
+    scorecardProgress() {
+      // progress value (0-100) to be passed to the Scorecard component
+      const totalAttempted = this.numOfCorrect + this.numOfWrong;
+      if (totalAttempted == 0) return 0;
+      return (this.numOfCorrect / (this.numOfCorrect + this.numOfWrong)) * 100;
+    },
+    scorecardMetrics() {
+      // define all the metrics to show in the scorecard here
+      return [
+        {
+          name: "numberOfCorrect",
+          icon: {
+            source: "check.svg",
+            color: "text-green-500",
+          },
+          value: this.numOfCorrect,
+          description: this.$t("player.scorecard.metric.description.correct"),
+        },
+        {
+          name: "numberOfWrong",
+          icon: {
+            source: "times-solid.svg",
+            color: "text-red-500",
+          },
+          value: this.numOfWrong,
+          description: this.$t("player.scorecard.metric.description.wrong"),
+        },
+        {
+          name: "numberOfSkipped",
+          icon: {
+            source: "hrline.svg",
+            color: "text-yellow-700",
+          },
+          value: this.numOfSkipped,
+          description: this.$t("player.scorecard.metric.description.skipped"),
```

Comment:
as before, I think instead of description, it should be `heading` or `title` or `name`